spec/feature/help_cookies_spec.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.describe "HelpCookies", type: :feature do
+  context "rendering the cookies setting page" do
+    before do
+      payload = { base_path: "/help/cookies", format: "special_route", title: "Cookies on GOV.UK", description: "You can choose which cookies you're happy for GOV.UK to use." }
+      stub_content_store_has_item("/help/cookies", payload)
+    end
+
+    it "renders the cookies setting page correctly" do
+      visit "/help/cookies"
+
+      within("#content") { expect(page).to have_css("h1", text: "Cookies on GOV.UK") }
```

Comment:
Minor: `expect(page).to have_title("Cookies on GOV.UK")`

Reply:
Done!